### PR TITLE
Merge 0.6 docs into master

### PR DESCRIPTION
**Pull request by justinxzhao**

### CODE REVIEW

**Comment by arnavgarg1** [comment]
@justinxzhao Is the plan to merge for now (even though its partially incomplete) and then continue just directly adding to master as needed for anything that's left over the next week or so?

**Comment by justinxzhao** [comment]
@arnavgarg1 that's what I'm thinking

**Comment by arnavgarg1** [comment]
Cool sounds good, I think that makes sense.

File: docs/user_guide/datasets/dataset_zoo.md [hunk 1]
```diff
@@ -88,8 +88,10 @@ train_stats, _, _ = model.train(training_set=training_set, test_set=test_set, mo
 Here is the list of the currently available datasets:
 
 | Dataset                                   | Hosted On             | Description                                                                                      |
-|-------------------------------------------|-----------------------|--------------------------------------------------------------------------------------------------|
+| ----------------------------------------- | --------------------- | ------------------------------------------------------------------------------------------------ |
 | adult_census_income                       | archive.ics.uci.edu   | <https://archive.ics.uci.edu/ml/datasets/adult>. Whether a person makes over $50K a year or not. |
+| allstate_claims_severity                  | Kaggle                | <https://www.kaggle.com/c/allstate-claims-severity>                                              |
+| amazon_employee_access_challenge          | Kaggle                | <https://www.kaggle.com/c/amazon-employee-access-challenge>                                      |
```
Comment (arnavgarg1) [inline docs/user_guide/datasets/dataset_zoo.md:94]: Do we know if these are the only new ones that have landed in 0.6? I know @dantreiman has been adding a lot of new datasets to Ludwig so just wanted to make sure. Can deal with this in a follow up PR, not a blocker in any way.

**Review by arnavgarg1 — APPROVED** [review]
Looks good to me! Might want to tally any remaining documentation updates so we know what still needs to be merged.

File: docs/user_guide/datasets/dataset_zoo.md (hunk 1 above)
Comment (justinxzhao) [reply docs/user_guide/datasets/dataset_zoo.md:94]: Looks like all of the datasets from https://github.com/ludwig-ai/ludwig-docs/pull/210/files are present.

**Pull request merged**